src/libsync/syncfileitem.h

```diff
@@ -284,6 +284,8 @@ class OWNCLOUDSYNC_EXPORT SyncFileItem
     bool _isRestoration BITFIELD(1); // The original operation was forbidden, and this is a restoration
     bool _isSelectiveSync BITFIELD(1); // The file is removed or ignored because it is in the selective sync list
     EncryptionStatus _e2eEncryptionStatus = EncryptionStatus::NotEncrypted; // The file is E2EE or the content of the directory should be E2EE
+    EncryptionStatus _e2eEncryptionMaximumAvailableStatus = EncryptionStatus::NotEncrypted;
```

Comment:
what does maximum stands for ?
I fail to understand

Reply:
@mgallien this depends on the `api-version` for encryption on the current server, if the `api-version` is `1.2` - we can not encrypt using version `2.0`, hence, we keep this information here in this member

Reply:
OK, so maybe use `_e2eEncryptionServerCapability` ?